torchtune/modules/layer_norm.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from typing import Any
+
+from torch import nn, Tensor
+
+
+class Fp32LayerNorm(nn.LayerNorm):
```

Comment:
sounds good. I will remove it

Reply:
do you know why we keep rmsNorm and the float32 logic instead of using torchs native?

torchtune: https://github.com/pytorch/torchtune/blob/main/torchtune/modules/rms_norm.py
torch: https://pytorch.org/torchtune/stable/generated/torchtune.modules.RMSNorm.html